<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdaterDialog</class>
 <widget class="QDialog" name="UpdaterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firmware/Software Updater</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="1" column="0">
    <widget class="QPushButton" name="startFirmware">
     <property name="text">
      <string>Start Firmware Update...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="startSoftware">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Start Software Update...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="messageArea"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
